examples/2d-meta_train.py

```diff
@@ -0,0 +1,87 @@
+import pytz
```

Comment:
[Minor] Nice tool but `pytz` is not included in RL4CO's dependence packages. Maybe better to have a package check here: 

```python
try:
    import pytz
except ImportError:
    # raise a warning and use python default timeit
    pass
```

Reply:
Yeah, maybe it could be removed.